lib/babel-istanbul-instrumenter.js

```diff
@@ -0,0 +1,256 @@
+/**
```

Comment:
Just to be clear, having this without `babel-istanbul-instrumenter` will add a whole lot of value by itself. Since we're now using the `preprocessTree` hook and `esprima` can inherently parse ES6, we get line numbers that map correct for ES6. The trouble comes only with ES7 features that we have gotten use to using like `decorators`.